legacy/src/api/payment-requests/payment-requests.md

```diff
@@ -458,6 +459,15 @@ Payment Activities are created when a Payment Request has been **created**, **pa
       "amount": "6190",
       "assetType": "epay.nzd.test",
       "productCodes": [ "23403" ]
+    },
+    {
+      "amount": "6190",
+      "assetType": "centrapay.token.test",
+      "collections: [
+        {
+          "id": "345224"
```

Comment:
Do we need to return the collection name or image so that may be displayed in the front end?

Reply:
I don't think so as we don't do it for epay

Reply:
The brand img etc is more for displaying in the app